src/js/tabs/osd.js

```diff
@@ -1181,6 +1181,21 @@ OSD.loadDisplayFields = function() {
             positionable: true,
             preview: "#9876",
         },
+        UP_DOWN_REFERENCE: {
+            name: 'UP_DOWN_REFERENCE',
+            text: 'osdTextElementUpDownReference',
+            desc: 'osdDescUpDownReference',
+            defaultPosition() {
```

Comment:
This should match https://github.com/betaflight/betaflight/pull/10417/files#diff-d16e9162351448c73bbcb31a6707d14b6bedd751f02526c965d41027f325ec72R366

Reply:
I thought we were using a constant default position in the firmware now?